intellij/src/saros/intellij/ui/util/SafeDialogUtils.java

```diff
@@ -192,7 +213,20 @@ public static Integer showYesNoDialog(Project project, final String message, fin
         },
         ModalityState.defaultModalityState());
 
-    return response.get();
+    Integer result = response.get();
+
+    if (result == null) {
```

Comment:
I guess this can only happens if there is an exception. Is this exception properly thrown when calling invokeAndWait or is it just swallowed inside the runnable ?

Reply:
> Is this exception properly thrown when calling invokeAndWait or is it just swallowed inside the runnable ?

It is re-thrown to the caller. So this check is unnecessary. Will remove it.